carbon-design-system/carbon
# fix(uishell): close HeaderMenu on blur

Closes #4930 

{{short description}}

Close the HeaderMenu on blur, to prevent multiple HeaderMenus from being open at once 

**Changed**

If the selected element that triggered the blur has the attribute of `href=#`, then it's another menu, so the expanded HeaderMenu should be closed

#### Testing / Reviewing

1. `cd packages/react && yarn storybook`
2. go to http://localhost:9000/iframe.html?id=ui-shell--header-base-w-navigation&viewMode=story#
3. expand `Link 4`
4. select `Link 3`
- notice the expanded menu will close, this doesn't happen currently on http://react.carbondesignsystem.com/iframe.html?id=ui-shell--header-base-w-navigation&viewMode=story#
